CMakeLists_files.cmake

```diff
@@ -112,6 +112,7 @@ list (APPEND TEST_SOURCE_FILES
 if(Boost_VERSION_STRING VERSION_GREATER 1.53)
 	list(APPEND TEST_SOURCE_FILES
 	  tests/cpgrid/disjointPatches_test.cpp
+	  tests/cpgrid/eclCentroid_test.cpp
```

Comment:
This file does not appear to be included in the PR.  Did you accidentally not `add` it to `git`?

Reply:
Probably. I confused  two branches. Let's me add the file. Thanks!